charts/testrunnerdashboard/templates/workloadidentity.yaml

```diff
@@ -0,0 +1,28 @@
+# This step requires Config Connector to be installed in the cluster.
+# The equivalent terraform-ap-modules/testrunner/dashboard.tf can be used
+# in place of this manifest if Config Connector is not enabled.
+{{- if .Values.secrets.gcpServiceAccount.workloadIdentity.enabled }}
```

Comment:
I am working on config connector for the new cluster, but not in a way that will support project-wide IAM just yet (that requires the most dangerous permissions on config connector itself, so we're not there yet). Let's go with the Terraform option for using workload identity (which I have specially enabled there), we do that a number of other places in our devops infra